### ISSUE TITLE
Feature Request - Turn off all Plots

### ISSUE THREAD

**Issue by jakebeal** [TASBE/TASBEFlowAnalytics#110]
_From @jegentile on November 22, 2017 16:18_

Given the charting issues with Octave 4.2, could we add a setting that will turn off all plotting functionality in TASBE?

_Copied from original issue: jakebeal/TASBEFlowAnalytics#11_

**Comment by jakebeal** [comment]
There is a kludge that can support this already, albeit not cleanly:

`TASBEConfig.set('testing.fakeFigureSaves',true)`

With this setting (normally used for regression testing), the plots will still be created, but they will not be saved, which is where the problems are occurring, if I remember correctly.

**Comment by jegentile** [comment]
I am getting an error when Octave tries to create a graphics handle so an error occurs when the plot is being drawn. Could there be option to bypass all plotting?

```
error: __goaxes__: unable to create graphics handle
error: called from:
    axes at line 66, column 10
    gca at line 58, column 9
    newplot at line 148, column 8
    image at line 110, column 11
    smoothhist2D at line 95, column 5
    GMMGating at line 125, column 5
```

**Comment by jakebeal** [comment]
Now that I know this problem exists: yes, we can add an option for bypassing all plotting. Is this currently a blocker? If so, I will raise it to Triage 0.

**Comment by jegentile** [comment]
I've commented out all the plotting functions locally but we should have this option available for the container builds so we are pulling from the this TASBE repository and not my local fork.

**Comment by jakebeal** [comment]
Got it; Triage 0.  @lisatiberio please take note

**Comment by jakebeal** [comment]
Motivating problem resolved by Octave 4.2.1
